apps/omg_performance/lib/omg_performance/byzantine_events/generators.ex

```diff
@@ -0,0 +1,103 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Performance.ByzantineEvents.Generators do
+  @moduledoc """
+  Module to generate users,
+  and stream of transaction, utxo position, block using data from child chain
+  """
+
+  alias OMG.Eth
+  alias OMG.Eth.RootChain
+  alias OMG.State.Transaction
+  alias OMG.Utils.HttpRPC.Client
+  alias OMG.Utxo
+
+  require Utxo
+
+  @child_chain_url Application.get_env(:byzantine_events, :child_chain_url)
+
+  def generate_users(size, opts \\ [initial_funds: trunc(:math.pow(10, 18))]) do
+    async_generate_user = fn _ -> Task.async(fn -> generate_user(opts) end) end
+
+    1..size
+    |> Enum.chunk_every(10)
+    |> Enum.map(fn chunk ->
+      Enum.map(chunk, async_generate_user)
```

Comment:
let's not nest piping operators, as this is controversial. Split a `defp` out from this

Reply:
this?